distribution/test/itests/pom.xml

```diff
@@ -78,7 +80,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.11</version>
+                <version>${junit.version}</version>
```

Comment:
@tbatie do you recall what was broken? I completely removed hamcrest and junit and successfully ran all the itests but that was in master not on the branch here.